frameworks/kafka/src/main/dist/svc.yml

```diff
@@ -86,6 +86,8 @@ pods:
       {{/TASKCFG_ALL_SECURITY_KERBEROS_ENABLED}}
 
           # setup-helper determines the correct listeners and security.inter.broker.protocol.
+          # it relies on the task IP being stored in MESOS_CONTAINER_IP
+          ./bootstrap -resolve=false -template=false
```

Comment:
As a medium-term change we may want to make it more explicit as to what `bootstrap` is doing.

Reply:
Yes. I always forget that Go don't set parent envvars. Drat.